docs/en/latest/plugins/fault-injection.md

```diff
@@ -298,99 +156,64 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
 }'
 ```
 
-Test plugin：
+## Example usage
```

Comment:
Example or Example usage?

Reply:
We have enabled the Plugin and now how do we use it? What is its use case? So, an "Example usage".